hardware/camera/README.md

```diff
@@ -73,6 +73,12 @@ Full camera software documentation can be found [here]](../../raspbian/applicati
 | Triggers | Keypress, UNIX signal, timeout |
 | Extra modes | demo, burst/timelapse, circular buffer, video with motion vectors, segmented video, live preview on 3D models |
 
+## HQ Camera IR Filter Transmission Information
+
+The HQ Camera uses a Hoya CM500 Infrared filter. It's transmission characteristics are as represented in the following graph.
```

Comment:
For consistency, the heading should not be capitalised after the first word (except for the part that is the product name).
Infrared -> infrared
it's -> its